app/Http/Controllers/BaseAPIController.php

```diff
@@ -112,7 +112,7 @@ protected function listResponse($query)
              $query->whereHas('client', $filter);
         }
 
-        if (! Utils::hasPermission('view_all')) {
+        if (! Utils::hasPermission('admin')) {
```

Comment:
- Is there a 'view all' option
- In the future I assume we'd want to support per-entity settings

Reply:
There is no view all permissions, I did not consider the impact on the API when fleshing this out. ideally, we'd want to constrain the API output based on the 'view' permission. I'll have a look at this.